tutorials/Picklist.md

```diff
@@ -0,0 +1,60 @@
+Here is an overview about how to use the Picklist page object:
+
+    const PagePicklist = require('../../../src/components/Picklist/pageObject');
```

Comment:
since this is a doc example we need to use real paths not the internals so:
```const PagePicklist = require('react-rainbow-components/components//Picklist/pageObject');```